Doc/library/importlib.metadata.rst

```diff
@@ -170,6 +170,12 @@ the values are returned unparsed from the distribution metadata::
     >>> wheel_metadata['Requires-Python']  # doctest: +SKIP
     '>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*'
 
+``PackageMetadata`` also presents a ``json`` attribute that returns
```

Comment:
This is a Python 3.10 only feature and hence a versionadded directive can be added.

Slightly out of topic, are `Distribution` objects and `metadata` attribute immutable? Accessing json attribute from metadata objects repeats the computation though it looks like a computed once attribute so just curious if `metadata` attribute of Distribution, `json` attribute etc. can be made a cached_property.

Reply:
Fixed in 289665029e (also `versionchanged` for changes to the interpretation of the metadata).

Reply:
> can be made a cached_property?

Although technically the PackageMetadata object is mutable, the protocol for it doesn't allow for mutation, so it's effectively immutable, so yes, a `cached_property` could be employed. Such an optimization may be premature, so I'm slightly inclined to defer that concern until a use-case presents itself. I don't feel strongly about it though and would likely accept a PR to `importlib_metadata`.